### ISSUE TITLE
Using camera or webcam

### ISSUE THREAD

**Issue by doccatdev** [PyXploiter/Barcode-Detection-and-Decoding#5]
Hello, i want to ask, can this code be modify to using camera or webcam for detect and decode the barcode and show the result of barcode code ?

**Comment by nivb52** [comment]
@readdoc-png 
Yes, it can be done, and more easily you can capture the image using a webcam or  camera and save it to file and then use this (temp) image. You will use this capture image as argument.
